docs/detailed-design.md

```
@@ -464,7 +464,181 @@ STT設定はトップレベルに配置（プリセット共通）:
+  debug?: boolean
+}
+```
+
+- `cache: false`（デフォルト）: キャッシュチェックせず常に生成。ファイル名はハッシュ値、ログは追記。
+- `cache: true`: キャッシュがあればそれを返し、なければ生成してキャッシュ。
+
+### 12.4 ファイル名
+
+`/api/generate` のファイル名は `cache` の値に関係なく常にハッシュ値を使用:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)